components/hal/esp32/include/hal/i2c_ll.h

```diff
@@ -262,6 +262,19 @@ static inline void i2c_ll_set_tout(i2c_dev_t *hw, int tout)
     hw->timeout.tout = tout;
 }
 
+/**
+ * @brief  Enable I2C timeout
+ *
+ * @param  hw Beginning address of the peripheral registers
+ * @param  tout_en Set true to enable I2C timeout
+ *
+ * @return None
+ */
+static inline void i2c_ll_set_tout_en(i2c_dev_t *hw, bool tout_en)
+{
+    ;//ESP32 do not support
```

Comment:
better to use the `UNUSED` macro, or (void)(hw);

Reply:
I was just following the pattern in this file